apps/dotcom/client/src/tla/components/TlaAppMenu/TlaAppMenuContent.tsx

```diff
@@ -0,0 +1,251 @@
+import { getCurrentEditor } from '../../utils/getCurrentEditor'
+import { defineMessages, useMsg } from '../../utils/i18n'
+import { clearLocalSessionState } from '../../utils/local-session-state'
+import { SubmitFeedbackDialog } from '../dialogs/SubmitFeedbackDialog'
+import { TlaManageCookiesDialog } from '../dialogs/TlaManageCookiesDialog'
+
+const messages = defineMessages({
+	help: { defaultMessage: 'Help' },
+	importFile: { defaultMessage: 'Import file…' },
+	getHelp: { defaultMessage: 'User manual' },
+	legalSummary: { defaultMessage: 'Legal summary' },
+	terms: { defaultMessage: 'Terms of service' },
+	privacy: { defaultMessage: 'Privacy policy' },
+	cookiePolicy: { defaultMessage: 'Cookie policy' },
+	manageCookies: { defaultMessage: 'Manage cookies' },
+	about: { defaultMessage: 'About tldraw' },
+	submitFeedback: { defaultMessage: 'Send feedback' },
+	signOut: { defaultMessage: 'Sign out' },
+	// debug
+	appDebugFlags: { defaultMessage: 'App debug flags' },
+	langAccented: { defaultMessage: 'i18n: Accented' },
+	langLongString: { defaultMessage: 'i18n: Long String' },
+	langHighlightMissing: { defaultMessage: 'i18n: Highlight Missing' },
+})
+
+export function TlaAppMenuContent() {
+	const isSignedIn = useAuth().isSignedIn
+
+	return (
+		<>
+			<TldrawUiMenuGroup id="signed-in-help">
+				<UserManualMenuItem />
+				<GiveUsFeedbackMenuItem />
+			</TldrawUiMenuGroup>
+			<TldrawUiMenuGroup id="links">
+				<LegalSummaryMenuItem />
+				{isSignedIn && <CookieConsentMenuItem />}
+			</TldrawUiMenuGroup>
+			<TldrawUiMenuGroup id="settings-and-rare-actions">
+				<ColorThemeSubmenu />
+				<LanguageMenu />
+				<DebugMenuGroup />
+				<ImportFileActionItem />
+			</TldrawUiMenuGroup>
+			<TldrawUiMenuGroup id="signout">
+				<SignOutMenuItem />
+			</TldrawUiMenuGroup>
+		</>
+	)
+}
+
+function SignOutMenuItem() {
+	const auth = useAuth()
+
+	const trackEvent = useTldrawAppUiEvents()
+
+	const label = useMsg(messages.signOut)
+
+	const handleSignout = useCallback(() => {
+		auth.signOut().then(clearLocalSessionState)
+		trackEvent('sign-out-clicked', { source: 'sidebar' })
+	}, [auth, trackEvent])
+
+	if (!auth.isSignedIn) return
+	return (
+		<TldrawUiMenuGroup id="account-actions">
+			<TldrawUiMenuItem id="sign-out" label={label} readonlyOk onSelect={handleSignout} />
+		</TldrawUiMenuGroup>
+	)
+}
+
+function ColorThemeSubmenu() {
+	const editor = useMaybeEditor()
+	if (!editor) return null
+	return <ColorSchemeMenu />
+}
+
+function CookieConsentMenuItem() {
+	const { addDialog } = useDialogs()
+	return (
+		<TldrawUiMenuItem
+			id="cookie-consent"
+			label={useMsg(messages.manageCookies)}
+			readonlyOk
+			onSelect={() => {
+				addDialog({ component: () => <TlaManageCookiesDialog /> })
+			}}
+		/>
+	)
+}
+
+export function UserManualMenuItem() {
+	const openAndTrack = useOpenUrlAndTrack('main-menu')
+	return (
+		<TldrawUiMenuItem
+			id="give-us-feedback"
+			label={useMsg(messages.getHelp)}
+			iconLeft="bookmark"
```

Comment:
v confusing choice of icon here too, people have a strong association with the bookmark icon as a way to track content, not as a way to learn about things. Also visually it feels misaligned with the comment icon below. Are there any others that might work? info circle maybe.

Reply:
+1 to a different icon here

Reply:
moved to a cute book